src/mbgl/shader/circle_uniforms.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <mbgl/shader/uniforms.hpp>
+
+namespace mbgl {
+
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_radius);
```

Comment:
Where multiple shaders use the same uniform name, I've hoisted the definition to [a shared file](https://github.com/mapbox/mapbox-gl-native/blob/ca28e8319522829dc9315f4c94d92d5a759ceb9c/src/mbgl/shader/uniforms.hpp).